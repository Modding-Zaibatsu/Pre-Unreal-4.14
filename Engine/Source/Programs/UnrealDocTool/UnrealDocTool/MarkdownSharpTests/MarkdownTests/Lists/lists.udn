1. The space on the end of this line is a part of the test so do not delete it. 
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum auctor 
ornare hendrerit. Donec justo nulla, consectetur non sollicitudin id, tempor
eget massa. Suspendisse porta, tortor quis tincidunt vulputate, lorem massa
faucibus arcu, ut tempor mi orci in est. Aenean turpis dolor, sollicitudin eu
sollicitudin sed, rutrum nec ligula. Quisque auctor ipsum et odio ornare, eget
lacinia velit dignissim. Nulla facilisi. Donec at dolor non est tincidunt
lacinia. Vestibulum pulvinar varius justo non placerat.

    Nulla vitae nisl sit amet leo suscipit sollicitudin et in mi. Etiam egestas
    diam sit amet tortor rutrum, vitae interdum odio fringilla. Cras et arcu
	vel dolor aliquam tempor. Ut molestie interdum lectus. Morbi gravida turpis
	ut diam dignissim condimentum. Nulla auctor tincidunt leo, at rutrum massa
	blandit adipiscing. Morbi posuere magna sodales erat porta, quis eleifend
	mauris volutpat. Nam dictum semper metus, quis ullamcorper sem fringilla
	vitae. Sed at diam vitae nibh porta rhoncus id vel turpis. Donec bibendum
	purus quis massa tempor fringilla. Sed aliquet venenatis tristique. Nam id
	purus neque. Aenean at sodales leo.

    1. Mauris vel dignissim enim, et commodo lacus. Vestibulum ante ipsum
	    primis in faucibus orci luctus et ultrices posuere cubilia Curae; Nulla
		facilisi. Vestibulum id lorem varius, tincidunt eros sed, vulputate
		ligula. Pellentesque ut orci eu arcu scelerisque aliquam non in ligula.
		Curabitur et odio sed arcu vehicula varius. Ut aliquam augue nec nulla
		tincidunt fringilla. In lacinia orci eget ante posuere, non semper
		lectus luctus. Sed vitae tincidunt sapien. Fusce tempor vitae erat sit
		amet auctor. Morbi ullamcorper odio sed vehicula consectetur. Phasellus
		at magna tellus. Donec vitae neque lacus.
		
		*   Sed scelerisque vehicula ante, non venenatis nulla elementum ut.
		    Vivamus varius sapien non tortor sollicitudin, ut luctus orci
			viverra. Curabitur vitae vulputate tellus. Donec sit amet odio ac
			arcu fringilla pretium vitae ut elit. Maecenas venenatis porttitor
			erat, at euismod nulla egestas sit amet. Sed nec dolor id felis
			commodo posuere vel nec ligula. Vivamus vel ante tristique,
			vulputate eros vel, laoreet enim. Ut dapibus lobortis lorem eu
			pellentesque. Phasellus et augue quis urna venenatis placerat.
			Nulla facilisi. Maecenas scelerisque et tortor et sodales. Ut
			dictum a odio nec ornare. Ut iaculis feugiat lorem, quis lacinia
			orci blandit dictum. Suspendisse eu tempor ipsum. Proin ultricies
			tortor mollis convallis pharetra.
			
		*   Sed dapibus viverra fermentum. Pellentesque faucibus neque neque,
		    eget aliquet lectus blandit bibendum. Etiam hendrerit tellus in
			urna ultrices, ac sollicitudin lacus tempus. Nulla ut aliquam
			ipsum. Curabitur rutrum ut est quis semper. Integer non augue
			felis. Nulla facilisi. Integer nec eleifend nibh. Nullam a nunc
			augue. Donec rutrum leo vel metus consequat, eget porttitor orci
			volutpat. Curabitur porta tellus tortor, sed consequat turpis
			molestie a. Integer a magna erat.
			
1. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum auctor
    ornare hendrerit. Donec justo nulla, consectetur non sollicitudin id,
	tempor eget massa. Suspendisse porta, tortor quis tincidunt vulputate,
	lorem massa faucibus arcu, ut tempor mi orci in est. Aenean turpis dolor,
	sollicitudin eu sollicitudin sed, rutrum nec ligula. Quisque auctor ipsum
	et odio ornare, eget lacinia velit dignissim. Nulla facilisi. Donec at
	dolor non est tincidunt lacinia. Vestibulum pulvinar varius justo non
	placerat.
1. Nulla vitae nisl sit amet leo suscipit sollicitudin et in mi. Etiam egestas
    diam sit amet tortor rutrum, vitae interdum odio fringilla. Cras et arcu
	vel dolor aliquam tempor. Ut molestie interdum lectus. Morbi gravida turpis
	ut diam dignissim condimentum. Nulla auctor tincidunt leo, at rutrum massa
	blandit adipiscing. Morbi posuere magna sodales erat porta, quis eleifend
	mauris volutpat. Nam dictum semper metus, quis ullamcorper sem fringilla
	vitae. Sed at diam vitae nibh porta rhoncus id vel turpis. Donec bibendum
	purus quis massa tempor fringilla. Sed aliquet venenatis tristique. Nam id
	purus neque. Aenean at sodales leo.


- Item A
- Item B
- Item C


+ Item A
+ Item B
+ Item C


Definition list with empty element:
$ Term 1: Definition 1
$ Term 2: Definition 2
$ Term 3
$ Term 4: Definition 4
