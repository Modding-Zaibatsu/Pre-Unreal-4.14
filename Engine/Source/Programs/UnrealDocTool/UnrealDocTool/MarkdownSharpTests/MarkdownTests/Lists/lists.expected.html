<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
    <head>
    </head>
    <body>
        <ol>
            <li>
                <p>
                    The space on the end of this line is a part of the test so do not delete it.
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum auctor
                    ornare hendrerit. Donec justo nulla, consectetur non sollicitudin id, tempor
                    eget massa. Suspendisse porta, tortor quis tincidunt vulputate, lorem massa
                    faucibus arcu, ut tempor mi orci in est. Aenean turpis dolor, sollicitudin eu
                    sollicitudin sed, rutrum nec ligula. Quisque auctor ipsum et odio ornare, eget
                    lacinia velit dignissim. Nulla facilisi. Donec at dolor non est tincidunt
                    lacinia. Vestibulum pulvinar varius justo non placerat.
                </p>
                
                <p>
                    Nulla vitae nisl sit amet leo suscipit sollicitudin et in mi. Etiam egestas
                    diam sit amet tortor rutrum, vitae interdum odio fringilla. Cras et arcu
                    vel dolor aliquam tempor. Ut molestie interdum lectus. Morbi gravida turpis
                    ut diam dignissim condimentum. Nulla auctor tincidunt leo, at rutrum massa
                    blandit adipiscing. Morbi posuere magna sodales erat porta, quis eleifend
                    mauris volutpat. Nam dictum semper metus, quis ullamcorper sem fringilla
                    vitae. Sed at diam vitae nibh porta rhoncus id vel turpis. Donec bibendum
                    purus quis massa tempor fringilla. Sed aliquet venenatis tristique. Nam id
                    purus neque. Aenean at sodales leo.
                </p>
                
                <ol>
                    <li>
                        <p>
                            Mauris vel dignissim enim, et commodo lacus. Vestibulum ante ipsum
                            primis in faucibus orci luctus et ultrices posuere cubilia Curae; Nulla
                            facilisi. Vestibulum id lorem varius, tincidunt eros sed, vulputate
                            ligula. Pellentesque ut orci eu arcu scelerisque aliquam non in ligula.
                            Curabitur et odio sed arcu vehicula varius. Ut aliquam augue nec nulla
                            tincidunt fringilla. In lacinia orci eget ante posuere, non semper
                            lectus luctus. Sed vitae tincidunt sapien. Fusce tempor vitae erat sit
                            amet auctor. Morbi ullamcorper odio sed vehicula consectetur. Phasellus
                            at magna tellus. Donec vitae neque lacus.
                        </p>

                        <ul>
                            <li>
                                <p>
                                    Sed scelerisque vehicula ante, non venenatis nulla elementum ut.
                                    Vivamus varius sapien non tortor sollicitudin, ut luctus orci
                                    viverra. Curabitur vitae vulputate tellus. Donec sit amet odio ac
                                    arcu fringilla pretium vitae ut elit. Maecenas venenatis porttitor
                                    erat, at euismod nulla egestas sit amet. Sed nec dolor id felis
                                    commodo posuere vel nec ligula. Vivamus vel ante tristique,
                                    vulputate eros vel, laoreet enim. Ut dapibus lobortis lorem eu
                                    pellentesque. Phasellus et augue quis urna venenatis placerat.
                                    Nulla facilisi. Maecenas scelerisque et tortor et sodales. Ut
                                    dictum a odio nec ornare. Ut iaculis feugiat lorem, quis lacinia
                                    orci blandit dictum. Suspendisse eu tempor ipsum. Proin ultricies
                                    tortor mollis convallis pharetra.
                                </p>
                            </li>

                            <li>
                                <p>
                                    Sed dapibus viverra fermentum. Pellentesque faucibus neque neque,
                                    eget aliquet lectus blandit bibendum. Etiam hendrerit tellus in
                                    urna ultrices, ac sollicitudin lacus tempus. Nulla ut aliquam
                                    ipsum. Curabitur rutrum ut est quis semper. Integer non augue
                                    felis. Nulla facilisi. Integer nec eleifend nibh. Nullam a nunc
                                    augue. Donec rutrum leo vel metus consequat, eget porttitor orci
                                    volutpat. Curabitur porta tellus tortor, sed consequat turpis
                                    molestie a. Integer a magna erat.
                                </p>
                            </li>
                        </ul>
                    </li>
                </ol>
            </li>

            <li>
                <p>
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum auctor
                    ornare hendrerit. Donec justo nulla, consectetur non sollicitudin id,
                    tempor eget massa. Suspendisse porta, tortor quis tincidunt vulputate,
                    lorem massa faucibus arcu, ut tempor mi orci in est. Aenean turpis dolor,
                    sollicitudin eu sollicitudin sed, rutrum nec ligula. Quisque auctor ipsum
                    et odio ornare, eget lacinia velit dignissim. Nulla facilisi. Donec at
                    dolor non est tincidunt lacinia. Vestibulum pulvinar varius justo non
                    placerat.
                </p>
            </li>

            <li>
                <p>
                    Nulla vitae nisl sit amet leo suscipit sollicitudin et in mi. Etiam egestas
                    diam sit amet tortor rutrum, vitae interdum odio fringilla. Cras et arcu
                    vel dolor aliquam tempor. Ut molestie interdum lectus. Morbi gravida turpis
                    ut diam dignissim condimentum. Nulla auctor tincidunt leo, at rutrum massa
                    blandit adipiscing. Morbi posuere magna sodales erat porta, quis eleifend
                    mauris volutpat. Nam dictum semper metus, quis ullamcorper sem fringilla
                    vitae. Sed at diam vitae nibh porta rhoncus id vel turpis. Donec bibendum
                    purus quis massa tempor fringilla. Sed aliquet venenatis tristique. Nam id
                    purus neque. Aenean at sodales leo.
                </p>
            </li>
        </ol>
        <ul>
            <li>
                <p>
                    Item A
                </p>
            </li>

            <li>
                <p>
                    Item B
                </p>
            </li>

            <li>
                <p>
                    Item C
                </p>
            </li>
        </ul>
        <ul>
            <li>
                <p>
                    Item A
                </p>
            </li>

            <li>
                <p>
                    Item B
                </p>
            </li>

            <li>
                <p>
                    Item C
                </p>
            </li>
        </ul>
        
        <p>Definition list with empty element:</p>
        <dl>
            <dt>Term 1</dt><dd><p>Definition 1</p></dd>
            <dt>Term 2</dt><dd><p>Definition 2</p></dd>
            <dt>Term 3</dt><dd></dd>
            <dt>Term 4</dt><dd><p>Definition 4</p></dd>
        </dl>
    </body>
</html>
